src/main/java/homework1/Main.java

```diff
@@ -0,0 +1,17 @@
+package Homework1;
+
+public class Main {
+    public static void main(String[] args) {
+
+        for (String s : args) {
+            if (s.equals("error")) {
+                System.out.println("\u001b[31m" + "ALARM");
```

Comment:
You forgot a closing escape string. If you don't use it, all your output until closing the application will be red. 
final String RED_COLOR = "\u001B[31m";
final String CLOSE_COLOR = "\u001B[0m";